[ISSUE] Error when creating UC Managed table : Missing cloud file system scheme

### Configuration
```hcl
resource "databricks_table" "this" {
  name               = "test"
  
  catalog_name       = local.catalog_name
  schema_name        = local.schema_name
  
  table_type         = "MANAGED"
  storage_location   = ""
  data_source_format = "DELTA"

  column {
    name      = "id"
    position  = 0
    type_name = "INT"
    type_text = "int"
    type_json = "{\"name\":\"id\",\"type\":\"integer\",\"nullable\":true,\"metadata\":{}}"
  }
}
```

### Expected Behavior
Table is created

### Actual Behavior
Error: 
```
2022-03-03T13:09:13.489+0100 [ERROR] vertex "databricks_table.this" error: cannot create table: Missing cloud file system scheme with databricks_table.this
```
### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Terraform and provider versions

terraform v1.1.0
databricks provider v0.5.0

### Debug Output
```
databricks_table.this: Creating...
2022-03-03T13:09:13.017+0100 [INFO]  Starting apply for databricks_table.this
2022-03-03T13:09:13.017+0100 [DEBUG] databricks_table.this: applying the planned Create change
2022-03-03T13:09:13.018+0100 [INFO]  provider.terraform-provider-databricks_v0.5.0: Using directly configured basic authentication: timestamp=2022-03-03T13:09:13.018+0100
2022-03-03T13:09:13.018+0100 [INFO]  provider.terraform-provider-databricks_v0.5.0: Configured basic auth: host=https://***REDACTED***.cloud.databricks.com/, username=***REDACTED***, password=***REDACTED***, account_id=***REDACTED***: timestamp=2022-03-03T13:09:13.018+0100
2022-03-03T13:09:13.018+0100 [DEBUG] provider.terraform-provider-databricks_v0.5.0: POST /api/2.0/unity-catalog/tables {
2022-03-03T13:09:13.488+0100 [DEBUG] provider.terraform-provider-databricks_v0.5.0: 400 Bad Request {
2022-03-03T13:09:13.489+0100 [WARN]  provider.terraform-provider-databricks_v0.5.0: /api/2.0/unity-catalog/tables:400 - Missing cloud file system scheme https://docs.databricks.com/dev-tools/api/latest/unity-catalog.html#tables: timestamp=2022-03-03T13:09:13.489+0100
2022-03-03T13:09:13.489+0100 [WARN]  provider.terraform-provider-databricks_v0.5.0: /api/2.0/unity-catalog/tables:400 - Missing cloud file system scheme https://docs.databricks.com/dev-tools/api/latest/unity-catalog.html#tables: timestamp=2022-03-03T13:09:13.489+0100
2022-03-03T13:09:13.489+0100 [ERROR] vertex "databricks_table.this" error: cannot create table: Missing cloud file system scheme
│   with databricks_table.this,
│   on main.tf line 65, in resource "databricks_table" "this":
│   65: resource "databricks_table" "this" {
path=.terraform/providers/registry.terraform.io/databrickslabs/databricks/0.5.0/darwin_arm64/terraform-provider-databricks_v0.5.0 pid=63201
```

--- a/catalog/resource_table.go
+++ b/catalog/resource_table.go
@@ -38,7 +38,7 @@ type TableInfo struct {
 	TableType             string            `json:"table_type"`
 	DataSourceFormat      string            `json:"data_source_format"`
 	ColumnInfos           []ColumnInfo      `json:"columns" tf:"alias:column"`
-	StorageLocation       string            `json:"storage_location"`
+	StorageLocation       string            `json:"storage_location,omitempty"`
 	StorageCredentialName string            `json:"storage_credential_name,omitempty"`
 	ViewDefinition        string            `json:"view_definition,omitempty"`
 	Owner                 string            `json:"owner,omitempty" tf:"computed"`

